### DISCUSSION TITLE
Backpack LED Panel, no output

### DISCUSSION THREAD

**Discussion by x4FF3** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#691] [Q&A]
Hi, 

i was gifted following panel:

![image](https://github.com/user-attachments/assets/70942e27-cfe1-4245-99e9-21b4d0c1038b)
![image](https://github.com/user-attachments/assets/d84c0a56-e140-4a62-b1f9-12d93eca233f)
![image](https://github.com/user-attachments/assets/e0c103aa-9105-4321-8120-8ad4f0720463)
![image](https://github.com/user-attachments/assets/0ab6421b-c339-4cdc-aa21-595a3cb85b21)
![image](https://github.com/user-attachments/assets/1a3c9935-529b-47dc-b591-d61351d4f915)

with a link to a chinese hardware platform: https://oshwhub.com/shukkkk/hub75e_96_128

but regardless of my efforts, i cannot get an output. i tried all examples, but none does create any output. not even flickering. the driver should be regarding discussions here, shiftreg correct?

KR
david

**Comment by board707** [comment]
The panel seems to be quarter scan 96x128 32s. 
The library supports two scan panels only.

**Comment by x4FF3** [comment]
ok, i am just curious, because if you follow the link, he seems to got it working with this library.
